charts/datadog/README.md

```diff
@@ -595,6 +595,7 @@ helm install --name <RELEASE_NAME> \
 | datadog.podLabelsAsTags | object | `{}` | Provide a mapping of Kubernetes Labels to Datadog Tags |
 | datadog.processAgent.enabled | bool | `true` | Set this to true to enable live process monitoring agent |
 | datadog.processAgent.processCollection | bool | `false` | Set this to true to enable process collection in process monitoring agent |
+| datadog.processAgent.processDiscovery | bool | `true` | Set this to false to disable autodiscovery of integrations |
```

Comment:
Contradicts the actual default (`false`).

Reply:
Oh thanks for catching that, I forgot to update it. Should be fixed now